docs/pages/setup/guides/teleport-operator.mdx

```diff
@@ -1,17 +1,24 @@
 ---
-title: Teleport Kubernetes Operator
+title: Teleport Kubernetes Operator (Preview)
 description: Easily manage Teleport resources from Kubernetes
 ---
 
-The Teleport Kubernetes Operator provides a way for Kubernetes users to manage Teleport resources like users and
-roles as Kubernetes resources.
-
-<Admonition type="note">
-  Setting up the Teleport Operator is not required to
-  [deploy Teleport on Kubernetes](../helm-deployments.mdx), nor setup
-  [Kubernetes Access](../../kubernetes-access/introduction.mdx).
+<Admonition type="warning" title="Preview">
+  Teleport Kubernetes Operator is currently in Preview mode.
 </Admonition>
 
+The Teleport Kubernetes Operator provides a way for Kubernetes users to manage some Teleport resources through
+Kubernetes, following [the Operator Pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/).
+
+The Teleport Kubernetes Operator is deployed alongside its CustomResourceDefinitions. Once deployed, users
```

Comment:
```suggestion
The Teleport Kubernetes Operator is deployed alongside its custom resource definitions. Once deployed, users
```
Using the convention in the Kubernetes docs: https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/